### PR TITLE
always use migrated teacher resources for migrated units

### PR DESCRIPTION
**Pull request by davidsbailey**
Continues [PLAT-1680](https://codedotorg.atlassian.net/browse/PLAT-1680). See [work plan](https://docs.google.com/document/d/1yIL2Z7aqFTdbCi24-RpUhPO_B3wRdsiKRTB1EEs17EY/edit#heading=h.1l25aiwflbrc).

now that migrated teacher resources have had time to be translated, we no longer need to temporarily show unmigrated teacher resources on migrated units.

## screenshots

Screen Shot 2022-06-01 at 5 13 40 PM

## Testing story

manually verified that all teacher resources in CSF 2020 are translated into es-MX (see screenshot for example).

### CODE REVIEW

**Review by dmcavoy — APPROVED** [review]

File: apps/src/code-studio/components/progress/UnitOverviewTopRow.jsx [hunk 1]
```diff
@@ -126,8 +126,6 @@ class UnitOverviewTopRow extends React.Component {
       courseVersionId
     } = this.props;
 
-    const useMigratedTeacherResources = isMigrated && !teacherResources.length;
```
Comment (bethanyaconnor) [inline apps/src/code-studio/components/progress/UnitOverviewTopRow.jsx:129]: Do the teacher resources on the scripts need to be cleaned up?

**Review by bethanyaconnor — APPROVED** [review]
🎉

File: apps/src/code-studio/components/progress/UnitOverviewTopRow.jsx (hunk 1 above)
Comment (davidsbailey) [reply apps/src/code-studio/components/progress/UnitOverviewTopRow.jsx:129]: yes, the `teacher_resources` a.k.a. unmigrated teacher resources can be removed after this. I'll prep the PR for that now so we can review it while this topic is fresh in our minds.

Comment (davidsbailey) [reply apps/src/code-studio/components/progress/UnitOverviewTopRow.jsx:129]: https://github.com/code-dot-org/code-dot-org/pull/46651

**Pull request merged**